vector-im/element-android
# [Poll] History list: Load more UI mechanism (PSG-1095)

## Type of change


## Content


Implementing UX for loading the polls in the history list. Improving architecture to separate components into dedicated layers. This is only mocked data for now. Real fetching mechanism will be implemented later.

## Motivation and context


Part of #7864 

## Screenshots / GIFs



<img src="https://user-images.githubusercontent.com/46314705/212339551-7322647f-7796-4a0b-b983-ef6a35e0b211.gif" width=300 />

## Tests



- Open any room profile screen
- Tap the "Poll history" entry
- Check the UX behavior by pressing the "Load more" button

## Tested devices

- OS version(s): Android 11

## Checklist
